src/org/labkey/test/tests/LabkeyErrorPageTest.java

```diff
@@ -0,0 +1,134 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyC;
+import org.labkey.test.pages.LabkeyErrorPage;
+
+import java.util.List;
+
+@Category({DailyC.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 7)
+public class LabkeyErrorPageTest extends BaseWebDriverTest
+{
+    private static final String READER_USER = "reader_user@user.test";
+    private static final String PROJECT_NAME = "Labkey Error Page Test";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        LabkeyErrorPageTest init = (LabkeyErrorPageTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+        createUserWithPermissions(READER_USER, getProjectName(), "Reader");
+    }
+
+    @Test
+    public void testGeneralErrors()
+    {
+        String imageTitle = "notFound_error.svg";
+        log("Verifying error message with mothership controller and action");
+        beginAt(WebTestHelper.buildURL("test", "NotFound"));
+        LabkeyErrorPage errorPage = new LabkeyErrorPage(getDriver());
+
+        checker().verifyEquals("Incorrect error heading message","Oops! The requested page cannot be found.",
+                errorPage.getErrorHeading());
+        checker().verifyEquals("Incorrect error sub-heading message","404: page not found.",
+                errorPage.getSubErrorHeading());
+        checker().verifyTrue("Incorrect error image",errorPage.getErrorImage().contains(imageTitle));
```

Comment:
Use hamcrest assertions to provide more informative error messages.
```suggestion
        checker().verifyThat("Incorrect error image", errorPage.getErrorImage(), CoreMatchers.containsString(imageTitle));
```

Reply:
done